lib/connectors/memory.js

```diff
@@ -538,17 +538,20 @@ Memory.prototype.destroyAll = function destroyAll(model, where, callback) {
   }
   var cache = this.collection(model);
   var filter = null;
+  var count = 0;
   if (where) {
     filter = applyFilter({where: where});
     Object.keys(cache).forEach(function (id) {
       if (!filter || filter(this.fromDb(model, cache[id]))) {
+        count++;
         delete cache[id];
       }
     }.bind(this));
   } else {
+    count = Object.keys(cache).length;
     this.collection(model, {});
   }
-  this.saveToFile(null, callback);
+  this.saveToFile(count, callback);
```

Comment:
I propose that we wrap `count` into an object to keep it flexible for future extensions, for example,

> this.saveToFile({count: count}, callback);